docs/features/software-templates/writing-templates.md

```diff
@@ -4,8 +4,7 @@ title: Writing Templates
 description: Details around creating your own custom Software Templates
 ---
 
-Templates are stored in the **Software Catalog** under a kind `Template`. You
```

Comment:
Hmm I still think that this is an important callout. Do we have this mentioned anywhere else?

Reply:
Yeah its also here: https://backstage.io/docs/features/software-templates/adding-templates, I was kind of following it "from top to bottom" and noticed it repeated, but I suppose we could also have it here. WDYT? 👍

Reply:
ah ok - sorry yeah I lazily didn't check - nice!